src/Clerk.ts

```diff
@@ -133,7 +133,22 @@ export default class Clerk {
 
   // Middlewares
 
-  expressMiddleware({ onError }: MiddlewareOptions = {}) {
+  // defaultOnError doesn't do anything
+  // return an Error with statusCode if you want to halt the request chain
+  defaultOnError(error: Error) {
+    console.error(error.message);
+  }
+
+  // strictOnError returns an error with statusCode 401 so that express will halt the request chain
+  strictOnError(error: Error) {
```

Comment:
Will this replace 5xx errors from clerk with a 401 back to the browser?

Reply:
Yes, it currently doesn't discern between error cases.

Reply:
Think I'm going to define a few error types to tell these cases apart:

- No cookie
- No session
- Miscellaneous error

Will need to see how to bring the same functionality to the next middleware somehow.

Reply:
Implemented pass through of server errors